text/0148-sampling-probability.md

```diff
@@ -0,0 +1,209 @@
+   values.
+3. For metrics raw events: Exemplars should be extended with the 
+   `sample_count` field defined above.
+
+### Example: Dapper tracing
+
+Google's [Dapper](https://research.google/pubs/pub36356/) tracing
+system describes the use of sampling to control the cost of trace
+collection at scale.
+
+### Example: Statsd metrics
+
+A Statsd counter event appears as a line of text, for example a metric
+named `name` is incremented by `increment` using a counter event (`c`)
+with the given `sample_rate`.
+
+```
+name:increment|c|@sample_rate
+```
+
+For example, a count of 100 that was selected for a 1-in-10 simple
+random sampling scheme will arrive as:
+
+```
+counter:100|c|@0.1
+```
+
+Probability 0.1 leads to a `sample_count` of 10.  Assuming the sample
+was selected using an unbiased algorithm (as by a "fair coin"), we can
+interpret this event as probabilistically equal to a count of `100/0.1
+= 1000`.
+
+## Internal details
+
+The statistical foundation of this technique is known as the
+Horvitz-Thompson estimator ("A Generalization of Sampling Without
+Replacement From a Finite Universe", JSTOR 1952).  The
+Horvitz-Thompson technique works with _unequal probability sampling_
+designs, enabling a variety of techniques for controlling properties
+of the sample.  
+
+For example, you can sample 100% of error events while sampling 1% of
+non-error events, and the interpretation of `sample_count` will be
+correct.
+
+### Bias, variance, and sampling errors
+
+There is a fundamental tradeoff between bias and variance in
+statistics.  The use of unbiased sampling leads unavoidably to
+increased variance.
+
+Estimating sampling errors and variance is out of scope for this
+proposal.  We are satisfied with the unbiased property, which guarantees
+that the expected value of totals derived from the sample equals the
+true population totals.  This means that statistics derived from
+sample data in this way are always accurate, and that more data will
+always improve precision.
+
+### Non-probabilistic rate-limiters
+
+Rate-limiting stages in a telemetry collection pipeline interfere with
+sampling schemes when they operate in non-probabilistic ways.  When
+implementing a non-probabilistic form of rate-limiting, processors
+MUST set the `sample_count` to zero.
+
+### No Sampler configured
+
+When no Sampler is in place and all telemetry events pass to the
+output, the `sample_count` field SHOULD be set to 1 to indicate
+perfect representivity, indicating that no sampling was performed.
+
+### Applicability for tracing
+
+The sampling approach covered here dictates how to select root spans
+in a probabilistic way.  When recording root spans, the `sample_count`
+field should be set as described above.  The adjusted `sample_count`
+of the root span applies the trace, meaning the trace should be
+considered as representative of `sample_count` traces in the
+population.
+
+When non-root spans are recorded because they are part of an ongoing
```

Comment:
@jmacd It seems like most use cases discussed below support having non-root spans carry the same `sample_count` field as the trace. What do you think of just dropping the paragraph from this section?

Reply:
I dropped it!